pages/about.html

```diff
@@ -4,9 +4,62 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <title>About me</title>
 </head>
 <body>
-  
+  <div class="container">
+    <header>
+      <h1>My life </h1>
+    <nav>
+        <ul>
+          <div class="box">
+          <div><li><a class="btn" href="/pages/about.html">About me</a></li></div>
+          <div><li><a class="btn" href="/pages/portfolio.html">Portfolio</a></li></div>
+          <div><li><a class="btn" href="/pages/index.html">Home</a></li></div>
```

Comment:
Careful with your file paths! The `/pages/about.html` path does not actually work because it is an incorrect absolute path. Starting a path with a slash (/) will start the file search at the root directory of your entire file system (not your project).
To correct this, we can use a relative path instead. This will provide directions to the target file based on the current file's location.
In code this looks like: `about.html`. The two dots at the beginning signal to start searching in the parent directory of the current directory.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This comment applies to the images as well, none of them show up when the page is opened.